src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -329,23 +326,13 @@ RoundaboutType RoundaboutHandler::getRoundaboutType(const NodeID nid) const
         if (1 == roundabout_name_ids.size() &&
             0 == connected_names.count(*roundabout_name_ids.begin()))
             return RoundaboutType::Rotary;
-        else
-            return RoundaboutType::Roundabout;
     }
 
-    if (radius <= MAX_ROUNDABOUT_INTERSECTION_RADIUS)
```

Comment:
In cases like this it would be helpful IMHO to add some context to the code. Link to OSM and a few words should suffices. Best would of course be a cucumber scenario testing for this behavior.